src/formatDate.js

```diff
@@ -50,7 +50,51 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const from = fromFormat[3];
+  const to = toFormat[3];
+
+  const obj = {};
```

Comment:
```suggestion
  const fromDivider = fromFormat[3];
  const toDivider = toFormat[3];

  const dateObj = {};
```
please give more meaningful names to variables